proto/api/v2/compliance_scan_configuration_service.proto

```diff
@@ -107,13 +107,21 @@ message ComplianceReportSnapshot {
   bool is_download_available = 8;
 }
 
+message FailedCluster {
+  string cluster_id = 1;
+  string cluster_name = 2;
+  string reason = 3;
```

Comment:
There may be more than one reason for failure, so we may either make it `repeated` or we would concatenate multiple strings into one.

Reply:
That's a good point. I'm not sure if changing it will break the UI. Let's maybe do the changes in the API in a follow-up and I'll change the storage protos in #15201 to have an error list. WDYT?

Reply:
Follow-up sounds totally okay!